src/app/shared/truncatable/truncatable-part/truncatable-part.component.ts

```diff
@@ -67,10 +95,67 @@ export class TruncatablePartComponent implements OnInit, OnDestroy {
     this.sub = this.service.isCollapsed(this.id).subscribe((collapsed: boolean) => {
       if (collapsed) {
         this.lines = this.minLines.toString();
+        this.expand = false;
       } else {
         this.lines = this.maxLines < 0 ? 'none' : this.maxLines.toString();
+        this.expand = true;
+      }
+    });
+  }
+
+  ngAfterContentChecked() {
+    if (this.isBrowser) {
+      if (this.observer && this.observedContent) {
+        this.toUnobserve();
+      }
+      this.toObserve();
+    }
+  }
+
+  /**
+   * Function to get data to be observed
+   */
+  toObserve() {
+    this.observedContent = this.document.querySelectorAll('.content');
+    this.observer = new (this._window.nativeWindow as any).ResizeObserver((entries) => {
+       for (let entry of entries) {
+        if (!entry.target.classList.contains('notruncatable')) {
```

Comment:
Since there's no `else` to this `if`, perhaps you can get rid of it by changing the selector at line 119 to `.content:not(.notruncatable)`